index.html

```diff
@@ -6318,10 +6318,13 @@ <h5>Presentational Roles Conflict Resolution</h5>
 				<p>A graphical object that controls the scrolling of content within a viewing area, regardless of whether the content is fully displayed within the viewing area.</p>
 				<p>A scrollbar represents the current value and range of possible values via the size of the scrollbar and position of the thumb with respect to the visible range of the orientation (horizontal or vertical) it controls. Its orientation represents the orientation of the scrollbar and the scrolling effect on the viewing area controlled by the scrollbar. It is typically possible to add or subtract to the current value by using directional keys such as arrow keys.</p>
 				<p>Authors MUST set the <pref>aria-controls</pref> attribute on the scrollbar element to reference the scrollable area it controls.</p>
-				<p>Authors MUST set the <pref>aria-valuemin</pref>, <pref>aria-valuemax</pref>, and <pref>aria-valuenow</pref> attributes.  If missing, their implicit values follow the same rules as the <a href="https://www.w3.org/TR/html5/forms.html#range-state-%28type=range%29" title="Range state">HTML range input type</a>:</p>
+				<p>Authors MAY set <pref>aria-valuemin</pref> and <pref>aria-valuemax</pref> to indicate the minimum and maximum thumb position. Otherwise, their implicit values follow the same rules as the <a href="https://www.w3.org/TR/html5/forms.html#range-state-%28type=range%29" title="Range state">HTML range input type</a>:</p>
+				<ul>
+				    <li>If <code>aria-valuemin</code> is missing or not a <a href="#valuetype_number">number</a>, it defaults to 0 (zero).</li>
+				    <li>If <code>aria-valuemax</code> is missing or not a <a href="#valuetype_number">number</a>, it defaults to 100.</li>
+				</ul>
+				<p>Authors MUST set the <pref>aria-valuenow</pref> attribute to indicate the current thumb position. If this attribute is missing or an unexpected value, the repair techniques follow the same rules as the <a href="https://www.w3.org/TR/html5/forms.html#range-state-%28type=range%29" title="Range state">HTML range input type</a>:</p>
```

Comment:
As specified in the issue, we are removing the default value and resorting to optional browser fallbacks specified in section 9.2. This paragraph and the following list are worded as there is a default. Suggest removing the list and rewording the paragraph to capture both the author and browser requirements as follows:

>Authors MUST set the aria-valuenow attribute to a value that represents the current thumb position. If aria-valuenow is missing or has an unexpected value, browsers MAY implement the repair techniques specified in the section describing handling author errors in states and properties, which are equivalent to the repair techniques for the HTML range input type.

Finally, aria-valuemin and aria-valuemax need to be removed from the table in section 9.2.

Reply:
Thanks for the suggestion, I like this messaging suggestion. Made the update and moved the reqs to the fallback table (was light on details before, like `slider`).

Similarly to `slider`, #949 asks what to do with fallback techniques for supported but not required attrs in the table.

Reply:
Removed supported but not required attrs from the fallback table as discussed on the call